src/parser/jobs/mnk/modules/Revolution.tsx

```diff
@@ -0,0 +1,48 @@
+import {Plural, Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import {Event} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const MIN_POTENCY_LOSS = 150
+const MAX_POTENCY_LOSS = 250
+
+export class Revolution extends Analyser {
+	static override handle = 'cr'
+
+	@dependency private data!: Data
+	@dependency private suggestions!: Suggestions
+
+	private revolutions: number = 0
+
+	override initialise(): void {
+		this.addEventHook(
+			filter<Event>()
+				.source(this.parser.actor.id)
+				.type('action')
+				.action(this.data.actions.CELESTIAL_REVOLUTION.id),
+			() => this.revolutions++)
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	private onComplete(): void {
+		if (this.revolutions > 0) {
```

Comment:
Personally I'd tiered suggestion to avoid this, but kinda up to you.

Reply:
I think using `TieredSuggestion` is a bit misleading when the resulting `Suggestion` isn't actually tiered, so I think I'll keep this conditional for now. That being said, I wonder if it makes sense to add a `value` field to the current `Suggestion` class that simply acts as a boolean display as given severity or `IGNORED`, though I think that's definitely outside the scope of this particular PR.